mmv1/third_party/terraform/resources/resource_container_cluster.go.erb

```diff
@@ -3066,7 +3066,7 @@ func expandMaintenancePolicy(d *schema.ResourceData, meta interface{}) *containe
 	}
 	maintenancePolicy := l[0].(map[string]interface{})
 
-	if maintenanceExclusions, ok := maintenancePolicy["maintenance_exclusion"]; ok && len(maintenanceExclusions.(*schema.Set).List()) > 0 {
```

Comment:
Hey @edwardmedia, I'm having difficulty understanding how this fixes the issue. If the list of maintenance exclusions is 0, wouldn't the two `for` loops below do nothing?

Reply:
This is the case when the count from 2 to 0.  With `len(maintenanceExclusions.(*schema.Set).List()) > 0` in place, it passes the below section when it become 0